packages/bson-bench/test/unit/task.test.ts

```diff
@@ -77,6 +78,10 @@ describe('Task', function () {
           expect(task.testName).to.not.include(test.library);
           expect(task.testName).to.match(/bson|bson-ext/);
         });
+
+        it('collects the tags correctly', function () {
+          expect(task.tags).to.deep.equal(['test']);
```

Comment:
Since you aren't adding this to tasks anymore, it's no longer a correct assertion; I assume you were intending to delete this test in favor of the one further down?